src/sagas/application.js

```diff
@@ -172,6 +172,39 @@ function* loginUserWorker({ emailAddress, password }) {
   }
 }
 
+function* logoutUserWorker() {
+  try {
+    // first, check to see if we are logged in
+    const loggedIn = yield select(isLoggedIn);
+
+    if (loggedIn) {
+      localStorage.setItem('accessToken', null);
+      localStorage.setItem('expiration', null);
+    }
+
```

Comment:
Don't we need to clear the accessToken and expiration from redux state as well?

Reply:
logoutUserSuccess resets the 'authorization' state to initialState